scss/civicrm/contact/pages/_contact-report.scss

```diff
@@ -5,6 +5,10 @@
     height: auto;
   }
 
+  .crm-submit-buttons {
+    @extend %submit-buttons-civi;
```

Comment:
can we change this to be a mixin instead?

Reply:
Ok, I will convert this one.